vue.config.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     proxy: {
       // this is needed to prevent a CORS error when running locally
       '/local-keycloak-config-url/*': {
-        target: 'https://business-filings-dev.pathfinder.gov.bc.ca/businesses/edit/config/kc/',
+        target: 'https://business-filings-dev.apps.silver.devops.gov.bc.ca/business/config/kc/',
```

Comment:
This is the "internal" URL for Filings UI. If possible, we should specify the external URL here.
Can you please check if this works instead?
`https://dev.bcregistry.ca/business/config/kc/keycloak.json`

Reply:
sure, will test this out and update if it works

Reply:
Updated and confirmed https://dev.bcregistry.ca/business/config/kc/ works.